apps/prairielearn/src/pages/administratorQueries/administratorQueries.ts

```diff
@@ -14,16 +15,21 @@ router.get(
   asyncHandler(async (req, res, _next) => {
     const fileList = await fsPromises.readdir(queriesDir);
     const jsonList = _.filter(fileList, (f) => /\.json$/.test(f));
```

Comment:
I like to avoid [Lodash](https://lodash.com/docs/4.17.15) in cases where there's a native JS equivalent. In this case, you can use [`Array.prototype.filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) directly.

Reply:
Resolved in commit a1e7740.